.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,26 @@
-## Pull Request
+<!-- ✨ Thanks for making a pull request! -->
 
-### Description
+### *What* change does this PR introduce?
 
-Briefly describe the purpose of this pull request.
+<!-- 🖋️ Describe the changes this PR introduces to help us understand the context of this change. -->
 
-### Type of change
+### *Why* was this change needed?
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)\
-- [ ] Refactoring (no functional changes, no api changes)
-- [ ] Documentation content changes
+<!-- 🚫 If your PR closes an open issue, link your PR to the issue with `Closes #16` or `Fixes #16`, where #16 is the issue number. For example:
+- Closes #16
+- Fixes #42
+Please put the close or fix statement in a bullet point so that it renders the issue title as well. -->
 
-### Issue ticket number and link
+### Other information (screenshots)
 
-[DAH-](https://igit.atlassian.net/jira/software/projects/DAH/boards/4?selectedIssue=DAH-)
+<!-- 📸 If you have any, please provide screenshots that would give additional visual context. Other forms of information (links to resources) would also be welcome. -->
 
 ### Checklist
 
-- [ ] Self-reviewed my code prior to submission.
-- [ ] Tests have been added or updated (if applicable).
-- [ ] Documentation has been updated (if applicable).
-- [ ] Screenshots added (if applicable).
-- [ ] This PR is ready for review.
+<!-- 📋 Here's a checklist of what needs to be done before this PR is ready for review. Once you have completed this checklist, you may request a review, and we will get back to you shortly. -->
 
-### Screenshots
+- [ ] 📝 I have reviewed my code prior to submission
+- [ ] ❔ Issue number has been linked (if applicable)
+- [ ] 🧪 Tests have been added or updated (if applicable)
+- [ ] 📖 Documentation has been updated (if applicable)
+- [ ] 📸 Screenshots added (if applicable)
```

Comment:
**praise:** This is a good upgrade and addresses all the requirements of a PR template. I guess it would be fun to give it a spin and see how it performs. Furthermore, we could always update it as needed as we continue to make progress.

Reply:
Thanks! Yeah, I think we should try it out for a couple of PRs to see how well it flows.